tests/treelist/treelist-aggregates.html

```diff
@@ -257,9 +259,9 @@
                         </div>
                     </div>
                     <div class="k-grid-container">
-                        <div class="k-grid-content k-virtual-content">
```

Comment:
I think the `k-virtual-content` class is needed for virtualization and I am not sure we should remove it - could you please check?